Userland/Libraries/LibGfx/ICCProfile.h

```diff
@@ -59,6 +59,67 @@ using TagTypeSignature = DistinctFourCC<FourCCType::TagTypeSignature>;     // IC
+    TAG(BToD2Tag, 0x42324432 /* 'B2D2' */)                          \
+    TAG(BToD3Tag, 0x42324433 /* 'B2D3' */)                          \
+    TAG(calibrationDateTimeTag, 0x63616C74 /* 'calt' */)            \
+    TAG(charTargetTag, 0x74617267 /* 'targ' */)                     \
+    TAG(chromaticAdaptationTag, 0x63686164 /* 'chad' */)            \
+    TAG(chromaticityTag, 0x6368726D /* 'chrm' */)                   \
+    TAG(cicpTag, 0x63696370 /* 'cicp' */)                           \
+    TAG(colorantOrderTag, 0x636C726F /* 'clro' */)                  \
+    TAG(colorantTableTag, 0x636C7274 /* 'clrt' */)                  \
+    TAG(colorantTableOutTag, 0x636C6F74 /* 'clot' */)               \
+    TAG(colorimetricIntentImageStateTag, 0x63696973 /* 'ciis' */)   \
+    TAG(copyrightTag, 0x63707274 /* 'cprt' */)                      \
+    TAG(deviceMfgDescTag, 0x646D6E64 /* 'dmnd' */)                  \
+    TAG(deviceModelDescTag, 0x646D6464 /* 'dmdd' */)                \
+    TAG(DToB0Tag, 0x44324230 /* 'D2B0' */)                          \
+    TAG(DToB1Tag, 0x44324231 /* 'D2B1' */)                          \
+    TAG(DToB2Tag, 0x44324232 /* 'D2B2' */)                          \
+    TAG(DToB3Tag, 0x44324233 /* 'D2B3' */)                          \
+    TAG(gamutTag, 0x67616D74 /* 'gamt' */)                          \
+    TAG(grayTRCTag, 0x6B545243 /* 'kTRC' */)                        \
+    TAG(greenMatrixColumnTag, 0x6758595A /* 'gXYZ' */)              \
+    TAG(greenTRCTag, 0x67545243 /* 'gTRC' */)                       \
+    TAG(luminanceTag, 0x6C756D69 /* 'lumi' */)                      \
+    TAG(measurementTag, 0x6D656173 /* 'meas' */)                    \
+    TAG(metadataTag, 0x6D657461 /* 'meta' */)                       \
+    TAG(mediaWhitePointTag, 0x77747074 /* 'wtpt' */)                \
+    TAG(namedColor2Tag, 0x6E636C32 /* 'ncl2' */)                    \
+    TAG(outputResponseTag, 0x72657370 /* 'resp' */)                 \
+    TAG(perceptualRenderingIntentGamutTag, 0x72696730 /* 'rig0' */) \
+    TAG(preview0Tag, 0x70726530 /* 'pre0' */)                       \
+    TAG(preview1Tag, 0x70726531 /* 'pre1' */)                       \
+    TAG(preview2Tag, 0x70726532 /* 'pre2' */)                       \
+    TAG(profileDescriptionTag, 0x64657363 /* 'desc' */)             \
+    TAG(profileSequenceDescTag, 0x70736571 /* 'pseq' */)            \
+    TAG(profileSequenceIdentifierTag, 0x70736964 /* 'psid' */)      \
+    TAG(redMatrixColumnTag, 0x7258595A /* 'rXYZ' */)                \
+    TAG(redTRCTag, 0x72545243 /* 'rTRC' */)                         \
+    TAG(saturationRenderingIntentGamutTag, 0x72696732 /* 'rig2' */) \
+    TAG(technologyTag, 0x74656368 /* 'tech' */)                     \
+    TAG(viewingCondDescTag, 0x76756564 /* 'vued' */)                \
+    TAG(viewingConditionsTag, 0x76696577 /* 'view' */)
+
+#define TAG(name, id) constexpr TagSignature name { id };
```

Comment:
Should these be `constexpr inline`? We do this in other headers to avoid creating a copy in each translation unit that uses these.

Reply:
Oh, yeah! I thought contexpr implies inline, but that's only true if they're statics in a class. My bad, fixed. Thanks!